File: easy/152_age_distribution.c
#include <stdio.h>

int main(int argc, const char * argv[]) {
    FILE *file = fopen(argv[1], "r");
    char line[1024];
    int age;

    while (fgets(line, 1024, file)) {
        sscanf(line, "%d", &age);

        if (0 <= age && age <= 2)
            printf("Still in Mama's arms\n");
        else if (3 <= age && age <= 4)
            printf("Preschool Maniac\n");
        else if (5 <= age && age <= 11)
            printf("Elementary school\n");
        else if (12 <= age && age <= 14)
            printf("Middle school\n");
        else if (15 <= age && age <= 18)
            printf("High school\n");
        else if (19 <= age && age <= 22)
            printf("College\n");
        else if (23 <= age && age <= 65)
            printf("Working for the man\n");
        else if (66 <= age && age <= 100)
            printf("The Golden Years\n");
        else
            printf("This program is for humans\n");
    }

    return 0;
}

File: easy/152_age_distribution.py
import sys

for line in open(sys.argv[1]):
    age = int(line)

    if 0 <= age <= 2:
        print( "Still in Mama's arms")
    elif 3 <= age <= 4:
        print( 'Preschool Maniac')
    elif 5 <= age <= 11:
        print( 'Elementary school')
    elif 12 <= age <= 14:
        print( 'Middle school')
    elif 15 <= age <= 18:
        print( 'High school')
    elif 19 <= age <= 22:
        print( "College")
    elif 23 <= age <= 65:
        print( 'Working for the man')
    elif 66 <= age <= 100:
        print( 'The Golden Years')
    else:
        print( 'This program is for humans')
